.github/workflows/ci.yaml

```
@@ -10,16 +10,16 @@ jobs:
   cla-check:
     if: github.event_name == 'pull_request'
     name: CLA check
-    uses: clay-lake/chisel-releases/.github/workflows/cla-check.yaml@main
+    uses: canonical/chisel-releases/.github/workflows/cla-check.yaml@main
```

Call the workflow locally instead

```suggestion
    uses: ./.github/workflows/cla-check.yaml
```